sdks/python/apache_beam/io/gcp/bigtable_io_write.py

```diff
@@ -0,0 +1,171 @@
+      self.instance = self.client.instance(self.beam_options.instance_id)
+    if self.table is None:
+      self.table = self.instance.table(self.beam_options.table_id,
+                                       self._app_profile_id)
+
+    self.batcher = MutationsBatcher(self.table, flush_count=self.flush_count,
+                                    max_row_bytes=self.max_row_bytes)
+
+  def process(self, row):
+    self.written.inc()
+    self.batcher.mutate(row)
+
+  def finish_bundle(self):
+    return self.batcher.flush()
+
+  def display_data(self):
+    return {'projectId': DisplayDataItem(self.beam_options.project_id,
+                                         label='Bigtable Project Id'),
+            'instanceId': DisplayDataItem(self.beam_options.instance_id,
+                                          label='Bigtable Instance Id'),
+            'tableId': DisplayDataItem(self.beam_options.table_id,
+                                       label='Bigtable Table Id'),
+            'bigtableOptions': DisplayDataItem(str(self.beam_options),
+                                               label='Bigtable Options',
+                                               key='bigtableOptions'),
+           }
+
+
+class BigtableConfiguration(object):
+  """ Bigtable configuration variables.
+
+  :type project_id: :class:`str` or :func:`unicode <unicode>`
+  :param project_id: (Optional) The ID of the project which owns the
+    instances, tables and data. If not provided, will
+    attempt to determine from the environment.
+
+  :type instance_id: str
+  :param instance_id: The ID of the instance.
+
+  :type table_id: str
+  :param table_id: The ID of the table.
+  """
+
+  def __init__(self, project_id, instance_id, table_id,
+               app_profile_id=None):
+    self.project_id = project_id
+    self.instance_id = instance_id
+    self.table_id = table_id
+    self.credentials = None
+    self.app_profile_id = app_profile_id
+
+
+class BigtableWriteConfiguration(BigtableConfiguration):
+  """ BigTable Write Configuration Variables.
+
+  :type flush_count: int
+  :param flush_count: (Optional) Max number of rows to flush. If it
+    reaches the max number of rows it calls finish_batch() to mutate the
+    current row batch. Default is FLUSH_COUNT (1000 rows).
+
+  :type max_mutations: int
+  :param max_mutations: (Optional)  Max number of row mutations to flush.
+    If it reaches the max number of row mutations it calls finish_batch()
+    to mutate the current row batch. Default is MAX_MUTATIONS
+    (100000 mutations).
+
+  :type max_row_bytes: int
+  :param max_row_bytes: (Optional) Max number of row mutations size
+    to flush. If it reaches the max number of row mutations size it
+    calls finish_batch() to mutate the current row batch. Default is
+    MAX_ROW_BYTES (5 MB).
+
+  :type app_profile_id: str
+  :param app_profile_id: (Optional) The unique name of the AppProfile.
+  """
+
+  def __init__(self, project_id, instance_id, table_id,
```

Comment:
For the initial release, we don't need `flush_count` and `max_row_bytes`.  We can add them later, if there's a need.